src/styles/global.scss

```diff
@@ -141,3 +143,9 @@ body {
   color: white;
 
 }
+
+.App > div:first-child {
+ padding-bottom: 2.5rem;
```

Comment:
Again, why `rem` here? Does this, too, need to be responsive?

Reply:
yes, I believe it does